### PR TITLE
#51/fix open requirements

### PR DESCRIPTION
**Pull request by FredLoh**

### CODE REVIEW

File: src/main/java/com/uber/nanoscope/Main.kt [hunk 1]
```diff
@@ -103,7 +103,7 @@ private fun acceptConfirmation(warningMessage: String?): Boolean {
 
 abstract class VersionedHandler: Runnable {
 
-    override final fun run() {
+    final override fun run() {
```
Comment (Leland-Takamine) [inline src/main/java/com/uber/nanoscope/Main.kt]: Any reason for this reordering?

**Comment by Leland-Takamine** [comment]
Thanks for the contribution @FredLoh. Would you mind squashing your commits?

File: src/main/java/com/uber/nanoscope/Main.kt (hunk 1 above)
Comment (FredLoh) [reply src/main/java/com/uber/nanoscope/Main.kt]: Lint change, based on the style guide.

**Comment by FredLoh** [comment]
I think you can squash them when you merge? Happy to make the change if that's not the case.

**Comment by Leland-Takamine** [comment]
Yes - it's possible to squash when merging but it ends up polluting the commit history in a different way so we've disabled that functionality for this repo. Instead, we always use "Rebase and merge" and manage our commit history locally.

**Comment by FredLoh** [comment]
Ah, makes sense, will squash. Thanks!

**Pull request merged**